pyannote/metrics/base.py

```diff
@@ -247,6 +324,21 @@ def __iter__(self):
         for uri, component in self.results_:
             yield uri, component
 
+    def __add__(self, other):
+        cls = self.__class__
+        result = cls()
```

Comment:
Once `clone` becomes a method of `BaseMetric`, we would do:

```suggestion
        result = self.clone()
```